raster/raster-layer-url/README.md

```diff
@@ -1,8 +1,16 @@
-# Raster Layer URL
+# Raster layer URL
```

Comment:
Just to highlight, we're not using the parenteses here around 'URL', so whichever way we choose, perhaps we should keep it consistent.